openff/interchange/interop/gromacs/_import/_import.py

```diff
@@ -117,8 +119,16 @@ def from_files(top_file, gro_file, cls=GROMACSSystem) -> GROMACSSystem:
 
             system.molecules.append((molecule_name, number_of_copies))
 
+        elif current_directive == "UNDEFINED":
+            raise GROMACSParseError(
+                "Failed to begin parsing GROMACS topology file. Error processing line:"
+                f"\n\n{line}\n"
+                "If you think this topology file is "
+                "valid, please open an issue on GitHub.",
+            )
+
         else:
-            raise ValueError(f"Invalid directive {current_directive}")
```

Comment:
(not blocking) Changing the type of error raised is a slight behavior change, could be worth mentioning in the release notes, though it's an experimental function to begin with so not super important.

Reply:
Good point, I made `GROMACSParseError` inherit from this so existing code is unlikely to be affected